training/distributed_training/pytorch/model_parallel_v2/gpt-neox/smp-finetuning-gpt-neox-fsdp-tp.ipynb

```diff
@@ -833,17 +837,6 @@
     "    )"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Use the private preview image from ECR which contains all\n",
-    "# libraries and dependencies required for our SageMaker job.\n",
-    "# image = CONTAINER_IMAGE"
```

Comment:
nit: Can remove def for `CONTAINER_IMAGE` at the top.

Reply:
Done!